### ISSUE TITLE
cannot pull docker images

### ISSUE THREAD

**Issue by mg03** [mesosphere/playa-mesos#42]
Hi,
cannot pull docker images from docker hub

docker pull gliderlabs/alpine:latest
Pulling repository gliderlabs/alpine
2015/12/16 02:38:52 Could not reach any registry endpoint

**Comment by ssk2** [comment]
Hi @mg03 - it looks that repository exists but your client may have difficulty connecting to it. What version of the Docker client are you running? They recently deprecated their old API and if your client is older, it most likely won't work.

**Comment by mg03** [comment]
HI @ssk2- you are correct the docker client was old

udo docker version
Client version: 1.0.1
Client API version: 1.12
Go version (client): go1.2.1
Git commit (client): 990021a
Server version: 1.0.1
Server API version: 1.12
Go version (server): go1.2.1
Git commit (server): 990021a

But this is the version inside the playa-mesos vagrant box. Any idea when that will be updated?

**Comment by ssk2** [comment]
Ah. Unfortunately, I'm afraid we're not continuing to update this Vagrant image - it will be replaced by another variant soon. For now, you can follow the Docker instructions to update this manually. Sorry I can't help more!

**Comment by lingmann** [comment]
This is likely due to Docker EOL'ing support for Docker clients <= 1.5:
https://forums.docker.com/t/docker-pull-deprecation-for-for-v1-5-clients/5071

tldr; upgrade your docker client. ;-)

**Comment by mg03** [comment]
@ssk2 - np .. im updating the vagrant file using a script as of now. Once yall release the next one ill use it.

thanx
